app/src/main/java/org/dhis2/utils/customviews/OptionSetView.java

```diff
@@ -250,20 +250,11 @@ protected boolean isEditable() {
 
     public void setViewModel(SpinnerViewModel viewModel) {
         setLayoutData(viewModel.isBackgroundTransparent(), viewModel.renderType());
-        setOnSelectedOptionListener(new OnSelectedOption() {
-            @Override
-            public void onSelectedOption(String optionName, String optionCode) {
-                /*processor.onNext(
-                        RowAction.create(viewModel.uid(), isSearchMode ? optionName + "_os_" + optionCode : optionCode, true, optionCode, optionName, getAdapterPosition())
-                );
-                if (isSearchMode)
-                    viewModel.withValue(optionName);
-                clearBackground(isSearchMode);*/
```

Comment:
where is clearBackground done now? i think you need to create a method in the customview